### PR TITLE
Wagtail 5.0

### PR DESCRIPTION
**Pull request by nickmoreton**
# Update for use with Wagtail v5+

### There is a PR back to the authors repo here: https://github.com/ibrahimawadhamid/wagtail_blocks/pull/12

This adds new svg support via: https://github.com/wagtail-nest/wagtail-font-awesome-svg which seems to be a more up to date package. It also has upcoming work to follow the next Wagtail releases.

It also includes: https://github.com/ibrahimawadhamid/wagtail_blocks/pull/11

I have also updated the documentation to reflect some of the changes since wagtail v3+

https://github.com/ibrahimawadhamid/wagtail_blocks/pull/11 can be closed

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
+
+```bash
 pip install wagtail_blocks
 ```
-2- Add the following enteries to your settings.py in the INSTALLED_APPS section:
+
+1. Add the following enteries to your settings.py in the INSTALLED_APPS section:
+
 ```python
 'wagtailfontawesome',
```
Comment (victoriachan) [inline docs/installation.md]: should change to `'wagtailfontawesomesvg',`.

**Review by victoriachan — CHANGES_REQUESTED** [review]
Looks good in general. 

Just a tiny change needed on installation.md to change `wagtailfontawesome` to `wagtailfontawesomesvg`.

I have also tested locally on a wagtail 5 build, and it works fine.

File: docs/installation.md (hunk 1 above)
Comment (nickmoreton) [reply docs/installation.md]: @victoriachan Nicely spotted, thanks. I have updated it.

**Review by victoriachan — APPROVED** [review]
Look good!

**Comment by nickmoreton** [comment]
This branch has been merged by the maintainer so closing this PR

**Pull request closed without merging**